manifests/test-alert.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Are you aware of the `*-alert.json` files inside the `examples` directory? If you want to add this new one please put it into the same `examples` directory (you may need to rebase, the `examples` directory was added recently).